tests/pipelines/eks/small/upstream-load.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: eks-small-upstream-load-pipeline-cron
+spec:
+  schedule: "0 8 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: curl
+            image: curlimages/curl
+            args: ["curl", "-X", "POST", "--data", "{}", "el-eks-small-upstream-load-cron-listener.svc.cluster.local:8080"]
+          restartPolicy: Never
+---
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: TriggerBinding
+metadata:
+  name: eks-small-upstream-load-cron-binding
```

Comment:
nit: i'd avoid putting the type of the object in the name (i.e. binding). You can also omit `cron`, since we will only ever have one binding. 

I love the convention of concatting the folder names into the names of these objects.